### ISSUE TITLE
NullPointerException in java

### ISSUE THREAD

**Issue by Yanpas** [MicrosoftDocs/intellicode#86]
**Product and Version** [VS/VSCode]: 1.33.1
**OS Version** [macOS/Windows]: ubuntu 19.04
**IntelliCode Extension Version**: 1.1.5
**Target Platform or Language** [e.g. Node.js]: java

**Steps to Reproduce / Scenario:**
Original issue: https://github.com/eclipse/eclipse.jdt.ls/issues/1003

Project example:
build.gradle:
```groovy
plugins {
    id 'java'
}
apply plugin: 'java'

repositories {
    mavenCentral()
}

dependencies {
    implementation group: 'org.apache.hbase', name: 'hbase-client', version: '1.4.9'
}

jar {
    baseName = 'gs-gradle'
    version =  '0.1.0'
}
```

src/main/java/Hello.java:
```java
import org.apache.hadoop.hbase.client.*;

public class Hello {
    public void fooBarBak() {
        Result r = null;
        r.getColumnLatestC(null, null)
    }
    public static void main(String[] args) {
        new Hello().fooBarBak();
    }
}
```

stack:
```

[Error - 12:56:21] 22 апр. 2019 г., 12:56:21 Problem with codeComplete for file:///home/yan/pg/gradle_hello/src/main/java/Hello.java
null
java.lang.NullPointerException
	at com.microsoft.jdtls.intellicode.core.ast.MethodInvokeVisitor.addMethodInfo(MethodInvokeVisitor.java:153)
	at com.microsoft.jdtls.intellicode.core.ast.MethodInvokeVisitor.visit(MethodInvokeVisitor.java:59)
	at org.eclipse.jdt.core.dom.MethodInvocation.accept0(MethodInvocation.java:220)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2836)
	at org.eclipse.jdt.core.dom.ASTNode.acceptChild(ASTNode.java:2884)
	at org.eclipse.jdt.core.dom.ExpressionStatement.accept0(ExpressionStatement.java:136)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2836)
	at org.eclipse.jdt.core.dom.ASTNode.acceptChildren(ASTNode.java:2907)
	at org.eclipse.jdt.core.dom.Block.accept0(Block.java:128)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2836)
	at org.eclipse.jdt.core.dom.ASTNode.acceptChild(ASTNode.java:2884)
	at org.eclipse.jdt.core.dom.MethodDeclaration.accept0(MethodDeclaration.java:617)
	at org.eclipse.jdt.core.dom.ASTNode.accept(ASTNode.java:2836)
	at com.microsoft.jdtls.intellicode.core.ast.MethodInvokeVisitor.<init>(MethodInvokeVisitor.java:47)
	at com.microsoft.jdtls.intellicode.core.pythia.PythiaCompletionContext.getCalls(PythiaCompletionContext.java:179)
	at com.microsoft.jdtls.intellicode.core.pythia.PythiaCompletionProvider.provideCompletionItems(PythiaCompletionProvider.java:59)
	at com.microsoft.jdtls.intellicode.core.CompletionProvider.computeContentAssist(CompletionProvider.java:117)
	at com.microsoft.jdtls.intellicode.core.CompletionProvider.provideCompletionItems(CompletionProvider.java:60)
	at com.microsoft.jdtls.intellicode.core.CommandHandler.executeCommand(CommandHandler.java:57)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler$1.run(WorkspaceExecuteCommandHandler.java:151)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.jdt.ls.core.internal.handlers.WorkspaceExecuteCommandHandler.executeCommand(WorkspaceExecuteCommandHandler.java:141)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$3(JDTLanguageServer.java:439)
	at org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$33(JDTLanguageServer.java:835)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:479)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

**Comment by jkeech** [comment]
Thank for reporting this issue! @Eskibear, can you take a look?

**Comment by Eskibear** [comment]
Sure, the NPE is confirmed, and I'll provide a fix for it.

**Comment by Eskibear** [comment]
Fixed in `vscode-intellicode-java@0.3.13`, now it looks like below:

![image](https://user-images.githubusercontent.com/2351748/56795206-85986c80-6842-11e9-8ae5-a74337848e4f.png)
